### PR TITLE
cephfs: obsolete deprecated mds commands

### PR DESCRIPTION
**Pull request by batrick**
Fixes: http://tracker.ceph.com/issues/20596

Signed-off-by: Patrick Donnelly <pdonnell@redhat.com>

### CODE REVIEW

**Comment by renhwztetecs** [comment]
mds dump and fs dump 2 types are puzzling, remove the mds dump is wise

**Comment by batrick** [comment]
Delaying this until after #17463 is merged which removes `mds dump` dependency in `ceph.in`.

TODO: `src/pybind/ceph_volume_client.py` also needs to remove `mds dump` dependency.

**Comment by batrick** [comment]
TODO: remove instances of `mds set` from `/qa`.

File: doc/cephfs/administration.rst [hunk 1]
```diff
@@ -178,21 +179,18 @@ filesystem.
 Legacy
 ------
 
-The ``ceph mds set`` command is the deprecated version of ``ceph fs set``,
-from before there was more than one filesystem per cluster. It operates
-on whichever filesystem is marked as the default (see ``ceph fs
-set-default``.)
+These legacy commands are obsolete and no longer usable post-Luminous.
```
Comment (tchaikov) [inline doc/cephfs/administration.rst]: shall we also mention this change in `PendingReleaseNotes`?

**Comment by tchaikov** [comment]
#17463 is merged.

**Pull request merged**